src/components/InventoryDetail/OsModeLabel.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Icon, Label, Popover, PopoverPosition } from '@patternfly/react-core';
+import FontAwesomeImageIcon from '../FontAwesomeImageIcon';
+import { BundleIcon } from '@patternfly/react-icons';
+
+const OsModeLabel = ({ isImageMode = false }) =>
```

Comment:
This component has a few things that could be refactored. For example the `bodyContent`, header, icon and label text could be extracted into an `osModes` object, like for example `{ image: {icon: ..., headerContent: ...,bodyContent: ..., label: ...}, package: {...}`. 

This would allow to just pass in a `mode` prop with "image" or "package" and use it query the parts like `osModes[mode]` and allow avoid the ternary operator and just work with a single branch of component.